COLLECTION.md

```diff
@@ -6,6 +6,7 @@
 
 |  #  | Brand              | Type        | Name                        | Origin      | Flavour                     |
 |-----|--------------------|-------------|-----------------------------|-------------|-----------------------------|
+| 142 | [Jia ming]         | [Green]     | [West Lake Dragon Well]     | [China]     | Green tea                   |
```

Comment:
The problem is the way how you organize the links. Many teas have the same name, for example, `West Lake Dragon Well`,  that's not the tea someone owned. It is a common name for that type of tea.